templates/cart/mini-cart.php

```diff
@@ -67,12 +67,14 @@
 		?>
 	</ul>
 
-	<p class="woocommerce-mini-cart__total total"><strong><?php _e( 'Subtotal', 'woocommerce' ); ?>:</strong> <?php echo WC()->cart->get_cart_subtotal(); ?></p>
+	<p class="woocommerce-mini-cart__total total"><?php do_action( 'woocommerce_widget_shopping_cart_total' ); ?></p>
```

Comment:
Missing docblock here saying about calling `woocommerce_widget_shopping_cart_subtotal` and the priority, like we does for another actions like:

```php
<p class="woocommerce-mini-cart__total total"><?php 
    /**
     * woocommerce_widget_shopping_cart_total hook.
     *
     * @hooked woocommerce_widget_shopping_cart_subtotal - 10
     */
    do_action( 'woocommerce_widget_shopping_cart_total' );
?></p>
```